package/CHANGELOG

```diff
@@ -1792,7 +1793,7 @@ Changes
 04/10/17 kain88-de, fiona-naughton, richardjgowers, tyler.je.reddy, jdetle
          euhruska, orbeckst, rbrtdlg, jbarnoud, wouterboomsma, shanmbic,
          dotsdl, manuel.nuno.melo, utkbansal, vedantrathore, shobhitagarwal1612,
-         xiki-tempula, kash1102
+         xiki-tempula, kash1102, Akshay0724, bsipocz, jeiros, tone.bengtsen
```

Comment:
```suggestion
         xiki-tempula, kash1102, Akshay0724, bsipocz, jeiros, tbengtsen
```
I think the correct GH handle for Tone Bengtsen's contribution in ENCORE is @tbengtsen

Reply:
Note that this handle is already a member of the MDA org so I am pretty sure that this is the correct handle because we invited all ENCORE folks.